entgql/template.go

```diff
@@ -443,12 +443,15 @@ func orderFields(n *gen.Type) ([]*OrderTerm, error) {
 			})
 		}
 	}
+	previousEdgeHasThroughTable := false
 	for _, e := range n.Edges {
 		name := strings.ToUpper(e.Name)
 		switch ant, err := annotation(e.Annotations); {
 		case err != nil:
 			return nil, err
-		case ant.Skip.Is(SkipOrderField), ant.OrderField == "":
+		case ant.Skip.Is(SkipOrderField): // skip edges marked with the skip annotation.
+		case ant.OrderField == "": // skip edges without the order field annotation.
+		case previousEdgeHasThroughTable: // orderBy is done on the edge, not the through table, so skip
```

Comment:
Is this problematic in any cases? All the tests seem to pass for me.

Reply:
It seems that for an edge with a through table that the list of edges has both the edge name and the through edge name. They appear in order, so we can skip the second through edge orderBy.